content/general_docs/bibliography.bib

```diff
@@ -1,75 +1,75 @@
 % Encoding: UTF-8
 
-@Book{fmi2.0,
-  publisher = {Modelica Association, Linköping, Sweden},
-  title = {FMI 2.0: Functional Mock-up Interface for Model Exchange and Co-Simulation},
-  year = {2014-07},
+@techreport{FMIFunctionalMock,
```

Comment:
```suggestion
@misc{fmi2.0,
```

This breaks the citations, so should ideally be referenced as fmi2.0, especially when we need fmi3.0 as well going forward. Also seems to me that handling this as @misc, like all the other standards seems more conistent...